lib/WatchmanFileCache.js

```diff
@@ -0,0 +1,122 @@
+import fbWatchman from 'fb-watchman';
+
+const SUBSCRIPTION_NAME = 'import-js-subscription';
+
+const CACHE = new Set();
+
+// Flag to quickly check if the cache is enabled.
+let enabled = false;
+
+function subscribe({ client, fbWatch, relativePath }): Promise {
+  const subscription = {
+    // Match any `.js` file
+    expression: ['allof', ['match', '*.js*']],
```

Comment:
This should maybe include `['type', 'f']`

Instead of the glob matcher, I wonder if we can use `suffix` and if that would be better. Maybe this works? `['suffix', ['js', 'jsx']]`